integration-tests/integration_test.go

```diff
@@ -685,6 +775,41 @@ func (s *IntegrationTestSuiteBase) GetProcesses(containerID string) ([]ProcessIn
 	return processes, nil
 }
 
+func (s *IntegrationTestSuiteBase) GetEndpoints(containerID string) ([]EndpointInfo, error) {
+	if s.db == nil {
+		return nil, fmt.Errorf("Db %v is nil", s.db)
+	}
+
+	endpoints := make([]EndpointInfo, 0)
```

Comment:
Where is `EndpointInfo` defined? I think it might be missing from this PR

Reply:
Oops. I forgot to add it. It is there now.